nss-evening-cohort-14/gitsub-team-no-name
# Ew packages

Completed Packages MVP and stretch goals. Preparing to merge.

## Description
I have completed all MVP, search bar, and delete function.

## Related Issue
https://github.com/nss-evening-cohort-14/gitsub-team-no-name/issues/22

## Motivation and Context
To make sure the JS on the Packages page is functional and ready to merge with the other pages.

## How Can This Be Tested?
- git pull origin ew-packages
- type info in form and press ‘add’ to see a new packages card
- search for a card title in the search bar
- click the delete button to delete a packages card

## Screenshots (if appropriate):

## Types of changes

- [ X] New feature (non-breaking change which adds functionality)
